### ISSUE TITLE
Unicode completion only partially works

### ISSUE THREAD

**Issue by thojDufour** [JuliaEditorSupport/zed-julia#26]
When trying to use Unicode characters in the buffer, most of them works except for the subscripts.

`\^1<TAB>` will return `¹`
`\_1<TAB>` will return `\_1` instead of  `₁`

Due to the auto-completion not being case sensitive,
`\alpha<TAB>` will return `Α` (capital alpha) instead of `α`

**Comment by Pangoraw** [comment]
This seems to be a problem in zed https://github.com/zed-industries/zed/issues/10926

I cannot reproduce the problem with `\_1<tab> -> ₁`.

**Comment by thojDufour** [comment]
Indeed the issue  with `\_1<tab> -> ₁` appear to have been solved . It might also have been the language server crashing. It's happening quite often on my machine.
Good to know about the case sensitivity thanks.

**Comment by lehoff** [comment]
I have \alpha giving a choice of capital and regular alpha, but \_ instantly gives the error message:

```
Missing reference: _
```

so I cannot get to the subscripts.

Zed 0.161.2
Julia extension: v0.1.8

**Comment by piechologist** [comment]
> capitalization of Greek letters in Zed

Yes, the REPL does it right but there are two issues here we can't fix in this extension. 

For instance, type `\pi` and the LS suggests `\\Pi`, `\\pitchfork`, `\\pisces`, and `\\pi` in this order (you can watch the LSP logs in Zed). Zed seems to sort the list alphabetically/uppercase first (Pi, pi, pisces, pitchfork). So, even if LanguageServer.jl responded in the expected order Zed would mess that up.
